### PR TITLE
Feature 14 devices fleet management related fields

### LINKED ISSUE #14: Devices fleet management related fields

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #14

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -5830,6 +5848,19 @@ components:
...
           type: string
           description: Version of the application on Device
+        warrantyExpiry:
+          type: string
+          format: date-time
+          description: Date when the warranty expires
+        usageHours:
+          type: number
+          format: double
+          description: Number of hours the Device has been used
```
Comment (anttileppa) [inline swagger.yaml:5858]: Shouldn't this be readOnly?

File: swagger.yaml [hunk 2]
```diff
@@ -5717,6 +5717,24 @@ components:
           format: uuid
           readOnly: true
           description: Id of ExhibitionDeviceGroup that ExhibitionDevice belongs to
+          
+    MqttDeviceStatus:
+      type: object
+      description: |
+        MQTT notification message about devices status. This is actually not part of the REST API spec but a realtime notification via MQTT channel.
+        It is periodically sent by all devices to the server.
+      properties:
```
Comment (anttileppa) [inline swagger.yaml:5726]: I would imagine that deviceId, status and version would all be required

File: swagger.yaml [hunk 3]
```diff
@@ -5717,6 +5717,24 @@ components:
...
+        deviceId:
+          type: string
+          format: uuid
+          readOnly: true
```
Comment (anttileppa) [inline swagger.yaml:5730]: This can cause problems when constructing the model, are you sure that is should be readOnly?

**Review by anttileppa — APPROVED** [review]

File: swagger.yaml (hunk 1 above)
Comment (villejuutila) [reply swagger.yaml:5862]: Resolved.

File: swagger.yaml (hunk 2 above)
Comment (villejuutila) [reply swagger.yaml:5730]: Resolved.

File: swagger.yaml (hunk 3 above)
Comment (villejuutila) [reply swagger.yaml]: Resolved.

**Pull request merged**